backend/app/main.py

```diff
@@ -1084,6 +1086,32 @@ async def get_projects(
     return gerrydb_views
 
 
+@app.get("/api/document/{document_id}/thumbnail", status_code=status.HTTP_200_OK)
+async def get_thumbnail(
+    *,
+    document_id: str,
+    session: Session = Depends(get_session),
+):
+    try:
+        if thumbnail_exists(document_id):
+            return RedirectResponse(url=f"https://tilesets1.cdn.districtr.org/thumbnails/{document_id}.png")
+        else:
+            return RedirectResponse(url="/home-megaphone.png")
+    except:
+        return RedirectResponse(url="/home-megaphone.png")
+
+
+@app.post("/api/document/{document_id}/thumbnail", status_code=status.HTTP_200_OK)
+async def make_thumbnail(
+    *,
+    document_id: str,
+    background_tasks: BackgroundTasks,
+    session: Session = Depends(get_session),
+):
+    background_tasks.add_task(generate_thumbnail, session=session, document_id=document_id)
```

Comment:
Q: Are there any options to log errors with background tasks?

Reply:
From some googling, [the logger should work](https://stackoverflow.com/questions/75145424/fastapi-starlette-how-to-handle-exceptions-inside-background-tasks). I'll upgrade logger.info to `logger.error` calls for the existing sections